src/EditorFeatures/TestUtilities/Remote/InProcRemostHostClient.cs

```diff
@@ -29,10 +29,10 @@ public static async Task<RemoteHostClient> CreateAsync(Workspace workspace, bool
             var remoteHostStream = await inprocServices.RequestServiceAsync(WellKnownRemoteHostServices.RemoteHostService, cancellationToken).ConfigureAwait(false);
             var remotableDataRpc = new RemotableDataJsonRpc(workspace, inprocServices.Logger, await inprocServices.RequestServiceAsync(WellKnownServiceHubServices.SnapshotService, cancellationToken).ConfigureAwait(false));
 
-            var instance = new InProcRemoteHostClient(workspace, inprocServices, new ReferenceCountedDisposable<RemotableDataJsonRpc>(remotableDataRpc), remoteHostStream);
+            var current = $"VS ({Process.GetCurrentProcess().Id}) {Guid.NewGuid().ToString()}";
```

Comment:
Should a common helper be extracted for constructing the client id string (that can be shared between inproc and remote one)?

Reply:
sure.